dbtmetabase/parsers/dbt_folder.py

```diff
@@ -0,0 +1,196 @@
+import re
+import os
+import logging
+
+from pathlib import Path
+from typing import List
+
+import yaml
+
+from dbtmetabase.models.metabase import METABASE_META_FIELDS
+from dbtmetabase.models.metabase import MetabaseModel, MetabaseColumn
+
+
+class DbtFolderReader:
+    """
+    Reader for dbt project configuration.
+    """
+
+    def __init__(self, project_path: str):
+        """Constructor.
+
+        Arguments:
+            project_path {str} -- Path to dbt project root.
+        """
+
+        self.project_path = (
+            project_path
+            if project_path[0] != "~"
+            else (os.path.expanduser("~") + project_path[1:])
+        )
```

Comment:
I didn't really get this part. Isn't `expanduser()` safe to be used in the entire string?
```
>>> os.path.expanduser('~/needs/expansion')
'/Users/brito/needs/expansion

>>> os.path.expanduser('/does/not/need/expansion')
'/does/not/need/expansion'
```

I think there's no need to check if there's `~` in the string, and you can apply in the entire path (no need to break it into two).